code/game/gamemodes/miniantags/slaughter/slaughter.dm

```diff
@@ -30,7 +30,7 @@
 	melee_damage_upper = 30
 	see_in_dark = 8
 	var/boost = 0
-	bloodcrawl = BLOODCRAWL_EAT
+	bloodcrawl = 2 //For some reason, the define doesn't work here.
```

Comment:
Don't do this. Move the `#define` up to `__DEFINES` instead so that it actually gets defined before this file gets read.